backend/src/zac/core/services.py

```diff
@@ -935,7 +935,7 @@ async def fetch_documents(zios: list, doc_versions: Optional[Dict[str, int]] = N
     return responses
 
 
-def update_document(url: str, file: UploadedFile, data: dict):
+def update_document(url: str, data: dict, request_kwargs: dict):
```

Comment:
I wouldn't let the request_kwargs abstraction leak that far, just asking of the audit line is sufficient, and then you can do the request_kwargs w/ headers building in this function instead.
```suggestion
def update_document(url: str, data: dict, audit_line: str):
```